blocks/query/index.php

```diff
@@ -43,6 +43,13 @@ function wp_curate_query_block_init(): void {
 	 */
 	$allowed_taxonomies = apply_filters( 'wp_curate_allowed_taxonomies', [ 'category', 'post_tag' ] );
 
+	/**
+	 * Filter the maximum number of posts that can be displayed in the Query block.
+	 *
+	 * @param integer $max_posts The maximum number of posts to display.
+	 */
+	$max_posts = apply_filters( 'wp_curate_max_posts', 5 );
```

Comment:
This should default to the previous default, which is 10:
```suggestion
	$max_posts = apply_filters( 'wp_curate_max_posts', 10 );
```